pkg/postgres/schema/billing_metrics.go

```diff
@@ -0,0 +1,42 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+
+package schema
+
+import (
+	"reflect"
+	"time"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+)
+
+var (
+	// CreateTableBillingMetricsStmt holds the create statement for table `billing_metrics`.
+	CreateTableBillingMetricsStmt = &postgres.CreateStmts{
+		GormModel: (*BillingMetrics)(nil),
+		Children:  []*postgres.CreateStmts{},
+	}
+
+	// BillingMetricsSchema is the go schema for table `billing_metrics`.
+	BillingMetricsSchema = func() *walker.Schema {
+		schema := GetSchemaForTable("billing_metrics")
+		if schema != nil {
+			return schema
+		}
+		schema = walker.Walk(reflect.TypeOf((*storage.BillingMetrics)(nil)), "billing_metrics")
+		RegisterTable(schema, CreateTableBillingMetricsStmt)
+		return schema
+	}()
+)
+
+const (
+	// BillingMetricsTableName specifies the name of the table in postgres.
+	BillingMetricsTableName = "billing_metrics"
+)
+
+// BillingMetrics holds the Gorm model for Postgres table `billing_metrics`.
+type BillingMetrics struct {
```

Comment:
As mentioned in the other PR I think we should consider making `cores` and `nodes` separate columns. Perhaps in addition to the serialized proto if we have to keep that for consistency / integration with existing tooling.